[{"name": "test.test_to_grayscale.ToGrayscale.test_blue", "status": "passed", "message": "", "passed": true, "points": ["p03-11.2"], "backtrace": []}, {"name": "test.test_to_grayscale.ToGrayscale.test_gray", "status": "passed", "message": "", "passed": true, "points": ["p03-11.1"], "backtrace": []}, {"name": "test.test_to_grayscale.ToGrayscale.test_green", "status": "passed", "message": "", "passed": true, "points": ["p03-11.2"], "backtrace": []}, {"name": "test.test_to_grayscale.ToGrayscale.test_main", "status": "failed", "message": "5 != 3 : Expected 3 calls to subplot or one to subplots", "passed": false, "points": ["p03-11.2"], "backtrace": ["  File \"C:\\Users\\isaul\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\unittest\\case.py\", line 58, in testPartExecutor\n    yield\n", "  File \"C:\\Users\\isaul\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\unittest\\case.py\", line 634, in run\n    self._callTestMethod(testMethod)\n", "  File \"C:\\Users\\isaul\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\unittest\\case.py\", line 589, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"C:\\Users\\isaul\\Documents\\intro_to_ml\\mooc-fi\\mooc-data-analysis-with-python-2024-2025\\part03-e11_to_grayscale\\test\\test_to_grayscale.py\", line 73, in test_main\n    self.assertEqual(psubplot.call_count, 3, msg=\"Expected 3 calls to subplot or one to subplots\")\n"]}, {"name": "test.test_to_grayscale.ToGrayscale.test_red", "status": "passed", "message": "", "passed": true, "points": ["p03-11.2"], "backtrace": []}]